README.md

```diff
@@ -4,10 +4,10 @@ The Container Storage Interface (CSI) Driver for IBM block storage systems enabl
 This is the official operator to deploy and manage IBM block storage CSI driver.
 
 Supported container platforms (and architectures):
-  - OpenShift v4.6 (x86, IBM Z, and IBM Power Systems)
   - OpenShift v4.7 (x86, IBM Z, and IBM Power Systems)
-  - Kubernetes v1.19 (x86)
+  - OpenShift v4.8 (x86, IBM Z, and IBM Power Systems)
   - Kubernetes v1.20 (x86)
+  - Kubernetes v1.21 (x86)
 
 Supported IBM storage systems:
   - IBM Spectrum Virtualize Family including IBM SAN Volume Controller (SVC) and IBM FlashSystem® family members built with IBM Spectrum® Virtualize (FlashSystem 5010, 5030, 5100, 5200, 7200, 9100, 9200, 9200R)
```

Comment:
IBM Spectrum® Virtualize (including FlashSystem 5x00, 7200, 9100, 9200, 9200R)

Reply:
but 5010 is not 5x00